cmd/mdatagen/metadata.yaml

```diff
@@ -103,3 +103,79 @@ metrics:
       aggregation: delta
     warnings:
       if_enabled: This metric is deprecated and will be removed soon.
+
+feature-gates:
```

Comment:
These feature gates should be added to the specific component's metadata.yaml files, not here. This one is just for testing purposes